csvqb/csvqb/tests/unit/utils/qb/test_standardise.py

```diff
@@ -0,0 +1,171 @@
+import pandas as pd
+import pytest
+from pandas.core.arrays.categorical import Categorical
+
+from csvqb.models.cube import *
+from csvqb.utils.qb.standardise import (
+    ensure_qbcube_data_is_categorical,
+    convert_data_values_to_uri_safe_values,
+)
+
+
+def test_qbcube_data_is_categorical():
+    """
+    Ensure that when we convert a QbCube's dataframe columns to categorical, it is done for the appropriate columns.
+    """
+    data = pd.DataFrame(
```

Comment:
Why do we convert QbCube’s DataFrame columns(appropriate columns) to Categorical data type?

Reply:
Because it allows us to do a more performant replacement of the column's values.